shopping-cart-room/src/main/java/com/handstandsam/shoppingapp/cart/ShoppingCartRoom.kt

```diff
@@ -0,0 +1,97 @@
+package com.handstandsam.shoppingapp.cart
+
+import android.content.Context
+import androidx.room.Room
+import com.handstandsam.shoppingapp.models.Item
+import com.handstandsam.shoppingapp.models.ItemWithQuantity
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.channels.ConflatedBroadcastChannel
+import kotlinx.coroutines.channels.ReceiveChannel
+import kotlinx.coroutines.launch
+
+
+class ShoppingCartRoom(appContext: Context) :
```

Comment:
the name of the class is confusing and makes me wonder what a "room" is. Is this a database?

Reply:
Thanks, renamed to `RoomShoppingCartDao`.  Many other changes as well.